swift/downgrades/04ef9ecaa9e06a587f252f147462861e3d32846a/downgrade.ql

```diff
@@ -0,0 +1,47 @@
+class Element extends @element {
+  string toString() { none() }
+}
+
+query predicate unspecified_elements(Element e, string property, string error) {
+  unspecified_elements(e, property, error)
+  or
+  error = "Removed parameter packs during the database downgrade" and
```

Comment:
It would imply we know which version it will land at?  🤔

Reply:
Hmm... fair. I just know that as a user, my reaction to "Removed parameter packs during the database downgrade" would be "_Why?_", so if there's something we can do to explain why it happens or what the user can do about it, that'd be great.

Not a blocking comment though, just a drive-by thought.

Reply:
Thanks for reminding me which way around downgrades and upgrades go!

Can we mention a commit hash then? Although maybe users are generally free to upgrade to the tip of `main`, so it doesn't matter? (With enterprise customers, they might find it easier to upgrade from e.g. 2.6 to 2.11 than all the way to 2.15, but I guess their symptoms would run an upgrade rather than a downgrade script.)

Reply:
Note that users are no likely to see this unless they start writing queries themselves and start looking for these unspecified elements, which they likely won't.

Reply:
Update the message nevertheless. I agree that this something no one will ever see. Shall we maybe put an easter egg to give away some GH swag if anyone ever finds these strings? 😆